jkube-kit/parent/pom.xml

```diff
@@ -568,6 +569,12 @@
         <version>${version.vertx-infinispan}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.ow2.asm</groupId>
+        <artifactId>asm</artifactId>
```

Comment:
is this really required ?

Reply:
@sunix I added it because of maven-enforcer plugin complains. New version of jnx comes with asm 9.x, the rest of the dependencies use older one.

Reply:
ok in any case we need a CQ for the 2 new/updated dependencies to merge this PR.